.github/workflows/uncrustify.yml

```diff
@@ -0,0 +1,33 @@
+name: "Uncrustify"
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  workflow_dispatch:
+
+jobs:
+  check:
+    name: Run format check using uncrustify
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Install uncrustify
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y uncrustify
```

Comment:
@jgromes Might be a good idea to install a specific version of uncrustify here, as version mismatch between the `uncrustify.cfg` and the one installed on the runner is what is causing the following error:

```
 Option<UNUM>: at /home/runner/work/RadioLib/RadioLib/extras/uncrustify/uncrustify.cfg:2838: Expected unsigned number , for 'align_nl_cont'; got 'false'
```

Latest uncrustify [expects an integer](https://jeayne.github.io/Uncrustify-Documentation/index_name.html) for the `align_nl_cont` option. However, it seems like this option used to accept a boolean (at least up till version [0.76.0](https://github.com/uncrustify/uncrustify/blob/5e1d2eccffae9ef5ddc6311dc01b9647af620994/etc/mono.cfg#L68)).

Reply:
@Dazza0 indeed, I will need to fix this to a specific version. Though this CI fail is due to the fact that [uncrustify found some issues](https://github.com/jgromes/RadioLib/actions/runs/16125371765/job/45500896401?pr=1532#step:4:35981), as I have yet to set the configuration to correctly cover all the corner cases. So it is working exactly as "expected".

Reply:
Done in https://github.com/jgromes/RadioLib/pull/1532/commits/d2c876a74314e2f0a0712d5ec4bd2f88d5ba92e0